{
    "type": "bundle",
    "id": "bundle--986d01b8-3e1d-499f-895b-60939c139846",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--12a23c85-74a5-4a2c-bff8-9203229e0138",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-01-19T00:29:23.323115Z",
            "modified": "2024-01-19T00:29:23.323115Z",
            "name": "CVE-2024-0654",
            "description": "A vulnerability, which was classified as problematic, was found in DeepFaceLab pretrained DF.wf.288res.384.92.72.22. Affected is an unknown function of the file mainscripts/Util.py. The manipulation leads to deserialization. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used. VDB-251382 is the identifier assigned to this vulnerability.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-0654"
                }
            ]
        }
    ]
}